nemo_skills/dataset/arena-hard-v2/prepare.py

```
@@ -0,0 +1,53 @@
+URL_BASELINE = (
+    "https://raw.githubusercontent.com/lm-sys/arena-hard-auto/main/data/arena-hard-v2.0/model_answer/o3-mini-2025-01-31.jsonl"
+)
+
+
+if __name__ == "__main__":
+    data_dir = Path(__file__).absolute().parent
+    data_dir.mkdir(exist_ok=True)
+    questions = str(data_dir / "question.jsonl")
+    baseline = str(data_dir / "o3-mini-2025-01-31.jsonl")
+    output_file = str(data_dir / "test.jsonl")
+    urllib.request.urlretrieve(URL_QUESTIONS, questions)
```

Ruff (0.14.10)

Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)